.github/workflows/build_and_deploy_ext2_img.yml

```diff
@@ -0,0 +1,133 @@
+name: Create and deploy ext2 image to pages
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+  # Lets you supply arguments when you run a github action manually.
+    inputs:
+      DOCKER_BUILD_CONTEXT:
```

Comment:
This is not what I intended. We should let people _only_ specify the Dockerfile name and deduce the context from it.

Input -> full path to Dockerfile
Context -> Directory where Dockerfile is

If the idea is still unclear, please try to elaborate on what it's missing